www/apps/resources/app/commerce-modules/product/guides/price-with-taxes/page.mdx

```diff
@@ -38,23 +38,27 @@ const taxModuleService = container.resolve(
 After resolving the resources, use Query to retrieve the products with the variants' prices for a context:
 
 ```ts
+import { QueryContext } from "@medusajs/modules-sdk"
+
+// ...
+
 const { data: products } = await query.graph({
-  entryPoint: "product",
+  entity: "product",
   fields: [
     "*",
     "variants.*",
     "variants.calculated_price.*",
   ],
-  variables: {
-    filters: {
-      id: "prod_123",
-    },
-    "variants.calculated_price": {
-      context: {
+  filters: {
+    id: "prod_123",
+  },
+  context: {
+    variants: new QueryContext({
```

Comment:
We don't need the `new` keyword and the `QueryContext` should encapsulate the `currency_code` and `region_id` instead of the entire `calculated_price`:
```
variants: { calculated_price: QueryContext({ currency_code: "usd", region_id: "region_123" }) }
```